### PR TITLE
Add CrossSiteTrackingPreventionRelaxedDomains

### PR DESCRIPTION
**Pull request by WardsParadox**
Adds new keys from 13.1, 16.2

https://developer.apple.com/documentation/devicemanagement/domains

### CODE REVIEW

File: Manifests/ManifestsApple/com.apple.domains.plist [hunk 1]
```diff
@@ -246,6 +246,48 @@ SafariPasswordAutoFillDomains definitions are cumulative. Patterns defined by al
...
+			</array>
+			<key>pfm_repetition_max</key>
+			<integer>10</integer>
+			<key>pfm_subkeys</key>
+			<array>
+				<dict>
+					<key>pfm_description</key>
+					<string></string>
+					<key>pfm_name</key>
+					<string>CrossSiteTrackingDomain</string>
```
Comment (apizz) [inline Manifests/ManifestsApple/com.apple.domains.plist:275]: If this is indeed an array of strings, `pfm_name` should be removed here.  `pfm_title` is the only thing you need besides the `pfm_type` and `pfm_value_placeholder`.  Remove the empty `pfm_description`

**Review by apizz — CHANGES_REQUESTED** [review]
Anytime you contribute, you should update the top-level `pfm_last_modified` date value and in this case where you're adding new preferences the `pfm_version` at the very bottom should also be bumped.

Other than that and the comment I made lgtm

File: Manifests/ManifestsApple/com.apple.domains.plist (hunk 1 above)
Comment (WardsParadox) [reply Manifests/ManifestsApple/com.apple.domains.plist]: Updating now

Comment (WardsParadox) [reply Manifests/ManifestsApple/com.apple.domains.plist]: Updated in 48cd1a0

**Review by relgit — APPROVED** [review]
Thanks for making the changes @WardsParadox.

**Comment by apizz** [comment]
Thanks again

**Review by apizz — APPROVED** [review]

**Pull request merged**